src/MSALWrapper/Locked.cs

```diff
@@ -64,5 +80,20 @@ public static T Execute<T>(ILogger logger, string lockName, TimeSpan maxLockWait
 
             return result;
         }
+
+        /// <summary>
+        /// Create a safe lockname.
+        /// </summary>
+        /// <param name="lockName">The lock name.</param>
+        /// <param name="visibility">The Mutex <see cref="Visibility"/>.</param>
+        /// <returns>A safe lock name derived from <paramref name="visibility"/> and <paramref name="lockName"/>.</returns>
+        public static string LockName(string lockName, Visibility visibility)

```

Comment:
@reillysiemens I see this method 'public' too instead of 'private'. I bring up because you had a comment on my PR for making methods private wherever possible. It's public here for unit tests I believe and works for me. Same reason I had public in my PR. Do let me know if I missed something in my PR. :)